python/lsst/rubintv/production/processingControl.py

```diff
@@ -1529,6 +1531,7 @@ def __init__(self) -> None:
         self._focalPlanePlot.showStats = False
         self._focalPlanePlot.plotMin = 0
         self._focalPlanePlot.plotMax = 1
+        self.IMAGING_IDS: tuple[int, ...] = tuple(detId for detId in self._imagingIds)
```

Comment:
I know you're just copying the pattern below so keeping it is fine for symmetry purposes but this is equivalent to `tuple(self._imagingIds)`